src/main/java/de/dennisguse/opentracks/util/StringUtils.java

```diff
@@ -63,6 +63,11 @@
         ISO_8601_BASE.setTimeZone(TimeZone.getTimeZone("UTC"));
     }
 
+    private static final long SECOND_IN_MILLIS = 1000;
```

Comment:
Should be in UnitConversions.

Reply:
I did all these changes to add datetime in track detail in a human friendly way, but I really don't like so much the way it looks.

Also, layouts will be changed so I'm going to remove all this code ;)